src/foam/lib/json/Outputter.java

```diff
@@ -265,6 +269,10 @@ protected Boolean maybeOutputProperty(FObject fo, PropertyInfo prop, boolean inc
     return true;
   }
 
+  protected  boolean propertyPredicate( FObject fo, PropertyInfo prop) {
+    return ( outputDefaultValues_ || prop.isSet(fo) );
```

Comment:
No need for parenthesis since there's no large expression with a higher priority.